scene/3d/camera.cpp

```diff
@@ -119,9 +119,11 @@ void Camera::_notification(int p_what) {
 		} break;
 		case NOTIFICATION_TRANSFORM_CHANGED: {
 
-			_request_camera_update();
-			if (doppler_tracking != DOPPLER_TRACKING_DISABLED) {
-				velocity_tracker->update_position(get_global_transform().origin);
+			if (current) {
```

Comment:
I think for cameras, this functionality should be transparent, i don't think it's a good idea to change anything in Camera.

Reply:
Yup, I believe this is all code from before, when I used Camera nodes rather than RIDs. I'll clean it up too.